modules/core/shared/src/test/scala/io/circe/JsonSuite.scala

```diff
@@ -0,0 +1,52 @@
+package io.circe
+
+import org.scalatest.flatspec.AnyFlatSpec
+
+class JsonSuite extends AnyFlatSpec {
```

Comment:
I'm new to this repo, and I was surprised to see there were no tests for the other functions in `modules/core/shared/src/main/scala/io/circe/Json.scala`

It looks like the tests for those functions are in [`circe/modules/tests/shared/src/test/scala/io/circe/JsonSuite.scala`](https://github.com/circe/circe/blob/master/modules/tests/shared/src/test/scala/io/circe/JsonSuite.scala)

I think you should move this test there too, and remove the library dependency declaration you added in `build.sbt`

Reply:
thanks! I was overlook `tests` project.
I did move my test code to `tests`.